backend/src/test/kotlin/org/loculus/backend/controller/submission/PreparedProcessedData.kt

```diff
@@ -99,6 +100,21 @@ val defaultProcessedDataMultiSegmented = ProcessedData(
     ),
 )
 
+val defaultProcessedDataWithoutSequences = ProcessedData<GeneticSequence>(
+    metadata = mapOf(
+        "date" to TextNode("2002-12-15"),
+        "host" to TextNode("google.com"),
```

Comment:
Isn't "host" the host of the virus, i.e. "human" or "mouse" or something? 😆 But a string is a string!

Reply:
Yes, that's a little easter egg ;)